packages/shared/ReactSymbols.js

```diff
@@ -17,7 +17,7 @@ export const REACT_PORTAL_TYPE: symbol = Symbol.for('react.portal');
 export const REACT_FRAGMENT_TYPE: symbol = Symbol.for('react.fragment');
 export const REACT_STRICT_MODE_TYPE: symbol = Symbol.for('react.strict_mode');
 export const REACT_PROFILER_TYPE: symbol = Symbol.for('react.profiler');
-export const REACT_PROVIDER_TYPE: symbol = Symbol.for('react.provider');
+export const REACT_CONSUMER_TYPE: symbol = Symbol.for('react.consumer');
 export const REACT_CONTEXT_TYPE: symbol = Symbol.for('react.context');
```

Comment:
This is the crux of the change.

Providers shifted from `REACT_PROVIDER_TYPE` (deleted) to `REACT_CONTEXT_TYPE`.

Consumers shifted from `REACT_CONTEXT_TYPE` to `REACT_CONSUMER_TYPE` (new).

This lets us avoid changing anything about `use(Foo)` or `contextType = Foo` and only change how we map a Fiber type to the underlying context.